packages/orbit-components/src/primitives/ButtonPrimitive/README.md

```diff
@@ -31,11 +31,13 @@ Table below contains all types of the props available in ButtonPrimitive compone
 | boxShadowActive  | `string`                   |            | The box-shadow of the ButtonPrimitive on active state.                                                                                                         |
 | boxShadowFocus   | `string`                   |            | The box-shadow of the ButtonPrimitive on focus state.                                                                                                          |
 | children         | `React.Node`               |            | The content of the ButtonPrimitive. [See Functional specs](#functional-specs).                                                                                 |
+| centered         | `boolean`                  | `false`    | Can only be used when `fullWidth` is true and if `iconLeft` and/or `iconRight` are defined. If `centered` prop is `true`, the Button will center everything.   |
 | circled          | `boolean`                  | `false`    | If `true`, the ButtonPrimitive will have circular shape.                                                                                                       |
 | contentAlign     | `string`                   | `center`   | The justify-content of inner container that wraps icons and the children.                                                                                      |
 | contentWidth     | `string`                   |            | The with of inner container that wraps the children.                                                                                                           |
 | dataTest         | `string`                   |            | Optional prop for testing purposes.                                                                                                                            |
 | id               | `string`                   |            | Set `id` for `ButtonPrimitive`                                                                                                                                 |
+| download         | `boolean \| string`        |            | Can only be used when `href` is defined. Adds the `download` attribute to the link.                                                                            |
```

Comment:
not really an issue, but might be better to be precise and change to `to the anchor element`, technically `link` is a completely different thing in HTML